admin_guide/out_of_resource_handling.adoc

```diff
@@ -60,6 +59,29 @@ When the node fails a pod, it terminates all containers in the pod, and
 the link:https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase[`PodPhase`] is transitioned to *Failed*.
 ====
 
+The node supports the `nodefs` and `imagefs` file system partitions when detecting disk pressure, as follows:
+
+* The `nodefs`, or `rootfs`, file system that the node uses for local disk volumes, daemon logs, emptyDir,
+and so on (for example, the file system that provides `/`). 
+* The `imagefs` file system that the container runtime uses for storing images and
+individual container-writable layers. Eviction thresholds are at 85% full for `imagefs`. The `imagefs` file system depends on the runtime and, in the case of Docker, which storage driver you are using:
```

Comment:
Re: Eviction thresholds are at 85% full for `imagefs`. 
Is this a static value or a changeable default value?

Do you have information about the differences based on runtime?